### PR TITLE
Hotfix/motors stop

### PR DESCRIPTION
**Pull request by DominikWawak**
---
 
## Basic Info
 
| Info | Please fill out this column |
| ------ | ----------- |
| Ticket(s) this addresses   | (Closes #IssueNumber) |
| Primary OS tested on | (Ubuntu, MacOS, Windows) |
| Drone platform tested on |  |
 
---
 
## Description of documentation updates required from your changes
 

---
 
## Screenshots (if appropriate):

### CODE REVIEW

File: psdk_wrapper/src/modules/telemetry.cpp [hunk 1]
```diff
@@ -834,6 +841,36 @@ PSDKWrapper::rc_callback(const uint8_t *data, uint16_t data_size,
...
+{
+  (void)data_size;
+  (void)timestamp;
+  std::unique_ptr<T_DjiFcSubscriptionEscData> esc_data =
+      std::make_unique<T_DjiFcSubscriptionEscData>(
+          *reinterpret_cast<const T_DjiFcSubscriptionEscData *>(data));
+  psdk_interfaces::msg::EscData esc_msg;
+  esc_msg.header.stamp = this->get_clock()->now();
+  // Populate the message with ESC data
+  for (int i = 0; i < 4; ++i)
```
Comment (biancabnd) [inline psdk_wrapper/src/modules/telemetry.cpp:856]: I think to make it as generic as possible, we should fill all the data available in here. Even though most cases will be quadcopters, as this wrappers intends to be as generic as possible, we should fill all the vector.

File: psdk_wrapper/include/psdk_wrapper/psdk_wrapper_utils.hpp [hunk 2]
```diff
@@ -39,6 +41,7 @@
 #define FLIGHT_STATUS_TOPICS_MAX_FREQ 50
 #define BATTERY_STATUS_TOPICS_MAX_FREQ 50
 #define CONTROL_DATA_TOPICS_MAX_FREQ 50
+#define ESC_DATA_TOPICS_FREQ 10
```
Comment (biancabnd) [inline psdk_wrapper/include/psdk_wrapper/psdk_wrapper_utils.hpp:44]: The max frequency that DJI gives you of this topic is actually 50 Hz.

**Review by biancabnd — COMMENTED** [review]
Nice addition @DominikWawak ! :confetti_ball: 
I left you some comments but overall, looks very good!

File: psdk_wrapper/cfg/psdk_params.yaml [hunk 3]
```diff
@@ -46,3 +46,4 @@
         flight_status: 1
         battery_level: 1
         control_information: 1
+        esc_data_frequency: 10
```
[the remark this answers is not in the document]
Comment (DominikWawak) [reply psdk_wrapper/cfg/psdk_params.yaml:49]: I wasn't sure how it will be applied for now to save on the bandwidth lets use 1 like you say, I agree

**Review by biancabnd — APPROVED** [review]

**Pull request merged**